docs/content/docs/contributing/Docs.md

```diff
@@ -343,11 +345,14 @@ Open **Preferences > Build, Execution, Deployment > Debugger**
 
 ## Changing documentation
 
-### How to check locally
+### How to check changes locally
 
-Run: `docs/local.sh`\
+Run `docs/local.sh`\
 It will open documentation at `localhost:1313`.\
-You can edit markdown files in the `content/` on the fly. Changes out of this directory usually require a restart.  
+You can edit markdown files in the `content/` on the fly. Changes out of this directory usually require a restart.
+
+By default, we don't publish [draft, future and expired](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content) pages.\
+To publish them, add relevant argument to the command: `docs/local.sh --buildDrafts`
```

Comment:
Sorry. I don't understand why we need support that. Can you write about scenarios?

Reply:
We don't add anything additional. It's a built-in feature that didn't work in this script.

My scenario:

- I want to add a new page but it's not completed yet: `draft: true`.
I've written it partially and don't want to show it to users.
- I commit this file as a draft. 
We discuss it in PR and have TODOs to further changes.
I can merge unfinished work.
- I do a final PR and "enable" this page to users.

Now, I need to create a google doc to discuss something unfinished. It's harder to check conflicts, we need a separate code review.

It's a complete analogy of feature-toggle for a page. Do changes step by step, and publish explicitly.
I don't want to use it for all pages. Only when we want to merge and fix in the next PR.